react/features/authentication/components/LoginDialog.native.js

```diff
@@ -281,7 +281,7 @@ class LoginDialog extends Component<Props, State> {
         if (conference) {
             r = dispatch(authenticateAndUpgradeRole(jid, password, conference));
         } else {
-            r = dispatch(connect(jid, password));
```

Comment:
Why this change? At a quick glance this might break authentication on some scenarios, no?

Reply:
No, the connect function doesn't accept any parameter

Reply:
https://github.com/jitsi/jitsi-meet/blob/7a677ead93d191f5dac5df909f3d948efbeec76e/react/features/base/connection/actions.native.js#L78 Isn't it this one?

Reply:
According to flow, not, but you're right. Thanks